### PR TITLE
Replace client.Client.{Put,Get}I with client.Client.{Put,Get}.

### PR DESCRIPTION
**Pull request by petermattis**
Isolate the usage of gob for storage of values to structured.Schema.

See #493.

### CODE REVIEW

**Comment by bdarnell** [comment]
LGTM

File: client/client_test.go [hunk 1]
```diff
@@ -309,40 +308,29 @@ func TestKVClientGetAndPutProto(t *testing.T) {
 	}
 }
 
-// TestKVClientGetAndPutGob verifies gets and puts of Go objects using the
-// KV client's convenience methods.
+// TestKVClientGetAndPut verifies gets and puts of using the KV
+// client's convenience methods.
 func TestKVClientGetAndPutGob(t *testing.T) {
```
Comment (spencerkimball) [inline client/client_test.go]: s/Gob//

**Comment by spencerkimball** [comment]
LGTM

File: client/client_test.go (hunk 1 above)
Comment (petermattis) [inline client/client_test.go]: Oops. Fixed.

**Pull request merged**